app/services/db/deleted_user/create.rb

```diff
@@ -0,0 +1,14 @@
+module Db
+  module DeletedUser
+    class Create
+      def self.call(user_id)
+        user = User.find_by(id: user_id)
+        return unless user
+        ::DeletedUser.create!(user_id: user.id,
+                              uuid: user.uuid,
+                              created_at: user.created_at,
```

Comment:
I think this is a misleading column name, WDYT of `user_created_at` or `user_original_created_at` because `created_at` often refers to this row, not the row of the other table

Reply:
Yes good idea.

Reply:
done